kubernetes.yaml

```diff
@@ -72,7 +72,7 @@ spec:
     spec:
       containers:
         - name: indexer
-          image: gcr.io/valora-inc/indexer:latest
+          image: gcr.io/valora-inc/indexer:a0f47727ce7abd43a1bcc87b48abecb22b8ef5ce
```

Comment:
Is this intended? So we should update this hash whenever we add any significant change, right?

Reply:
Correct -- I'll update the README.md or add a CONTRIBUTING.md to describe "how to cut a release".